dashboard-frontend/src/types/study.ts

```diff
@@ -1,31 +1,36 @@
+import { Currency } from './shared';
+
 export interface Study {
   id: string;
   name: string;
   slug: string;
-  createdAt: string;
+  createdAt: number;
```

Comment:
@nandanrao 
I changed this **createdAt** and the rest of the **datetime's** to be **timestamps** instead of using **ISO 8601**.

pd: I found this article https://www.moesif.com/blog/technical/timestamp/manage-datetime-timestamp-timezones-in-api/ that touch on the topic and it's a nice read 👍

Reply:
Hrm, seems like the blog says "use either", which seems reasonable to me, but yeah go for timestamps I'm fine with that!